apps/modernization-ui/src/components/button/Button.tsx

```diff
@@ -43,14 +43,14 @@ const Button = ({
             outline={outline}>
             {labelPosition === 'left' && children && icon ? (
                 <>
-                    {children}
+                    <span>{children}</span>
                     {icon}
                 </>
             ) : null}
             {labelPosition === 'right' && children && icon ? (
                 <>
                     {icon}
-                    {children}
+                    <span>{children}</span>
```

Comment:
Question: What were the `<span>`s for? If the `{children}` need specific styling shouldn't that be handled by whatever passes them in? Or does this fix a global styling issue where the `TrussworksButton` expects its direct child to be `display: inline`?